src/members/MemberProfile.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+import AuthProvider, { AuthContext } from "../AuthContext";
+import { Redirect } from "react-router-dom";
+
+
+export default function MemberProfile({member}) {
+
+  const { isAuth } = useContext(AuthContext);
```

Comment:
You don't need all this AuthContext here. /member-profile is also a protected route so all the auth happens. You don't need to repeat here.

Reply:
Ok. I've removed it in the latest commit